forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -223,6 +223,10 @@ def get():  # pylint: disable=too-many-locals
             form_type: str = dict_data.get("form_type", "form")
             auth_list = auth_form_details.get("authorizationList") or {}
             resource_list = [group["resourceId"] for group in auth_list]
+
+            if form_name:
```

Comment:
instead of replacing the special characters better to let the user know which characters are allowed for the form name

Reply:
formio allows all the characters in form name. 
our issue was in sending the special characters in the api call. 
we send the special characters (the entire form name) as encoded.
backend handles this to decode and get the expected result.

Reply:
In the backend, since we're using `alike` operator which has a different meaning for `%` and `_`.
So we replaced those 2 characters with `\`